ClinGen/clincoded
# Connects to #1161. Connects to #1154. Modify extra evidence module

### Connects to #1161:
Modified the extra evidence module to allow the user to see all interpretations also in Interpretation mode. As a result, the `ExtraEvidenceTableViewAll` class from the extra evidence module and references to it have been removed, and its code has been merged to the primary `ExtraEvidenceTable` class. Now, all extra evidence is visible in both Evidence View and Interpretation modes, with the user being able to add and modify extra evidence they've added in Interpretation mode. Extra evidence are sorted by last modified, and the Add PMID button has been moved to the top of the table.

As a result of this change, pages upstream within the VCI  had to be updated to update the `variant` object on extra evidence change, and send it downstream. The `session` object also had to be passed down to identify which extra evidence was relevant to the user.


Extra evidence table outside interpretation:
![image](https://cloud.githubusercontent.com/assets/4326866/21092640/7687e292-c002-11e6-9c87-73b64688dc79.png)

Extra evidence table within interpretation:
![image](https://cloud.githubusercontent.com/assets/4326866/21092624/5a12d5ea-c002-11e6-8945-2d795965dbab.png)

Testing:

1. Get to a VCI interpretation
2. Create extra evidence
3. Confirm that they are visible in Evidence View mode
4. Log in as different user and get to interpretation for same variant
5. Confirm that you cannot modify your extra evidence from other user
6. Add extra evidence and confirm you can edit your new extra evidence
7. Switch back to original user and confirm that you can edit your old extra evidence, but not the new ones created with the other user

### Connects to #1154:

Buttons have been restyled to use the wording 'Save', and have had their button colors changed to `btn-primary` from `btn-info`

![image](https://cloud.githubusercontent.com/assets/4326866/21092981/32965260-c004-11e6-913a-58a34091416f.png)

Testing:

1. Edit previously made variant object and confirm new button styling and wording
